docs/fundamentals/code-analysis/style-rules/ide0063.md

```diff
@@ -45,7 +45,46 @@ using var a = b;
 // csharp_prefer_simple_using_statement = false
 using (var a = b) { }
 ```
-
+### is using the new syntax a must?
+actually using "`using` new syntax, tells the compiler that the variable being declared should be disposed at the end of the enclosing scope" so if you have nothing after `using statement` but a closing curly bracket, IDE will suggest you to use new syntax otherwise you should think about it that if there is a difference between using new syntax or not (which in over 99% of times it won't)
```

Comment:
````suggestion
With the simple `using` statement, or `using` declaration, the variable is disposed at the end of the scope in which it's declared, which is usually at the end of the method. If there's nothing after the closing curly brace of the old-style `using` statement, IDE0063 is raised. However, if there are statements after the closing curly brace of an old-style `using` statement, IDE0063 may not be raised. This is because if you change to a `using` declaration, the variable will remain in scope for longer than it otherwise would; that is, it would remain in scope for the statements that appear after the closing curly brace of the original `using` statement. The following code snippet shows an example of such a method.

```csharp
private void ReadContact()
{
    using(SQLiteConnection conn = new(App.DbPath))
    {
        conn.CreateTable<Contact>();
        contacts = conn.Table<Contact>().ToList();
    }
    if (contacts.Count > 0)
    {
        contactsList.ItemsSource = contacts;
    }
}
```

````